jest-puppeteer.config.js

```diff
@@ -0,0 +1,22 @@
+const ci = Boolean(process.env.CI || false);
+const baseOptions = {
+  server: {
+    command: 'npm start',
+    port: 8000,
+  },
+};
+const ciPipelineOptions = {
+  launch: {
+    executablePath: '/usr/bin/google-chrome-stable',
+    headless: true,
+    args: [
+      '--ignore-certificate-errors',
+      '--no-sandbox',
+      '--disable-setuid-sandbox',
+      '--disable-accelerated-2d-canvas',
```

Comment:
I used these setting as a reference from the website which I refereed for setting jest-puppeteer. https://levelup.gitconnected.com/running-puppeteer-with-jest-on-github-actions-for-automated-testing-with-coverage-6cd15bc843b0 
On checking I found that - The option `--disable-accelerated-2d-canvas` in the `args` array disables hardware acceleration for the 2D canvas element in Google Chrome. This may be useful in some cases where hardware acceleration for the 2D canvas is causing issues or conflicts with other parts of the application.

Reply:
but do we need canvas? our browser is running in headless/mode (its not opening)

Reply:
removed